### ISSUE TITLE
AudioSwitch writes [Return] into keyboard buffer

### ISSUE THREAD

**Issue by yasuoiwakura** [sirWest/AudioSwitch#62]
hi there,

when calling AudioSwitch.exe with parameters, it writes somehow into the keyboard buffer.
when running from cmd.exe you can see that a second prompt appears before the actual output.

when calling AudioSwitch.exe by Eventghost (like i do) the suddenly stroke RETURN can cause very different behaviors.

for example, i bind Win+F9 to a Script wich runs AudioSwitch.exe:
i press Win+F9
i release Win+F9 (!)
small delay in Eventghost Script
EG calls AudioSwitch.exe
Windows gets the Win+RETURN input
Cortana tells me something weird sind Win+Return is bound to some windows function O_o

its really weird but not really related to EG, since you can see the RETURN behavior when simply running AudioSwitch.exe /?

**Comment by sirWest** [comment]
Yup, can't find a better working g solution to return back from running the program and return to cmd, if you know then would would help really to fix this.
